src/legacy/status_im/events.cljs

```diff
@@ -112,14 +111,14 @@
 (defn- on-biometric-auth-fail
   [{:keys [code]}]
   (if (= code "USER_FALLBACK")
-    (re-frame/dispatch [:multiaccounts.logout.ui/logout-confirmed])
+    (re-frame/dispatch [:profile.logout/logout])
     (utils/show-confirmation
```

Comment:
This rename is part of the removal of the `legacy...` namespace.


The new name was picked based on our current login events.